care/facility/api/serializers/daily_round.py

```diff
@@ -41,6 +41,27 @@ class DailyRoundSerializer(serializers.ModelSerializer):
 
     rounds_type = ChoiceField(choices=DailyRound.RoundsTypeChoice, required=True)
 
+    # Community Nurse's Log
+
+    bowel_issue = ChoiceField(
+        choices=DailyRound.BowelDifficultyType.choices, required=False
+    )
+    bladder_drainage = ChoiceField(
```

Comment:
Why do we need these fields in the serializer? It is already part of the model right?

Reply:
Without the serializer having the choices explicitly specified, front-end will have to reverse the choice text to it's integer equivalent when making the POST.

However, with the serializer explicitly specified, both are possible and that way front-end need not have to be concerned about what the integer equivalents are for each of the choice.

Here's an attempt with the serializer fields removed:

<img width="330" alt="image" src="https://github.com/user-attachments/assets/6d31cf99-2b62-48e3-87c4-70ac24eab6eb">


<img width="1706" alt="image" src="https://github.com/user-attachments/assets/a3f0ac8e-9dd6-4958-8f10-7e9386b80407">